src/ufw.yml

```diff
@@ -6,36 +6,46 @@
   roles:
     - ufw
   vars:
-    # Do not actually enforce firewall rules.  The assessors will turn
-    # UFW on if they want it.
-    state: disabled
+    # Enable ufw and enforce firewall rules by default.
+    state: enabled
```

Comment:
These lines can be removed.  `enabled` is [the default behavior](https://github.com/cisagov/ansible-role-ufw/blob/41e0944c9f2d17ce58e68812f1cc0386b6d32872/defaults/main.yml#L11-L13).

Reply:
Cleaned up in https://github.com/cisagov/teamserver-packer/pull/52/commits/52db4be9c4c96d0fd9851f6e94ecd30831bb74bf.